test.sh

```diff
@@ -42,26 +47,30 @@ function usage()
     exit 1
 }
 
-if [ "$#" -eq "0" ]; then
+if [ "$#" -eq "0" -o x"${1//-/}" == x"help" -o x"${1//-/}" == x"h" ]; then
     usage
 fi
 
+
 if [ -z "$CLUSTER_URL" ]; then
     echo "Cluster not found. Create and configure one then set \$CLUSTER_URL."
     exit 1
 fi
 
 if [ -z "$AWS_ACCESS_KEY_ID" -o -z "$AWS_SECRET_ACCESS_KEY" ]; then
-    echo "AWS credentials not found (\$AWS_ACCESS_KEY_ID and \$AWS_SECRET_ACCESS_KEY)."
-    exit 1
+    CREDENTIALS_FILE="$HOME/.aws/credentials"
+    if  [ -f "$CREDENTIALS_FILE" ]; then
```

Comment:
Yes. Most-likely. Suggestions? (apart from implementing a config file parser in bash).